javascript-timers/timeout.js

```diff
@@ -0,0 +1,4 @@
+
+setTimeout(function() {
+  document.querySelector('h1').innerHTML = 'Hello There';
```

Comment:
Typically it is better and safer to use `textContent` over `innerHtml`

Reply:
Oh okay thank you I will use that next time!